ztools/ztls/ztls_names.go

```diff
@@ -380,6 +381,14 @@ func init() {
 	cipherSuiteNames[0xFF83] = "SSL_RSA_WITH_3DES_EDE_CBC_MD5"
 	cipherSuiteNames[0xFF03] = "SSL_EN_RC2_128_CBC_WITH_MD5"
 	cipherSuiteNames[0xFF85] = "OP_PCL_TLS10_AES_128_CBC_SHA512"
+
+	// Name-value paires *are* not standardized, only dereferenced for JSON output
+	clientAuthTypeNames = make(map[int]string)
+	clientAuthTypeNames[0] = "NoClientCert"
+	clientAuthTypeNames[1] = "RequestClientCert"
+	clientAuthTypeNames[2] = "RequireAnyClientCert"
+	clientAuthTypeNames[3] = "VerifyClientCertIfGiven"
+	clientAuthTypeNames[4] = "RequireAndVerifyClientCert"
```

Comment:
Yes, these are Golang abstractions from ztls/common.go. The actual values were previously an iota and have not meaning other than to be part of an enum-style class.

![image](https://cloud.githubusercontent.com/assets/5420174/23343316/50b66a2c-fc37-11e6-815e-3c3ee527a23b.png)